build_tools/configure/configure_test.py

```diff
@@ -34,12 +34,20 @@
 
 # CUDA specific paths and versions
 _CUDA_SPECIFIC_PATHS_AND_VERSIONS = {
-    "cublas_version": "12.3",
-    "cuda_toolkit_path": "/usr/local/cuda-12.2",
+    "cuda_version": '"12.1.1"',
     "cuda_compute_capabilities": ["7.5"],
-    "cudnn_version": "8",
+    "cudnn_version": '"8.6"',
+    "ld_library_path": "/usr/local/nvidia/lib:/usr/local/nvidia/lib64",
+}
+_CUDA_COMPUTE_CAPABILITIES_AND_LD_LIBRARY_PATH = {
+    "cuda_compute_capabilities": [
+        "sm_50",
+        "sm_60",
+        "sm_70",
+        "sm_80",
+        "compute_90",
+    ],
     "ld_library_path": "/usr/local/nvidia/lib:/usr/local/nvidia/lib64",
```

Comment:
This directory pattern appears several times throughout this change.  Is it merely for symmetry with our nvidia/cuda dockerfiles?  (These paths are only used on very old nvidia-docker or on iGPU devices, to my knowledge.  There's no harm in including them; we do the same in our containers.  I'm just checking my understanding that there's not another purpose for having them here.)

Reply:
This particular file is just a unit test of a python script `configure.py`, I believe that the path `/usr/local/nvidia/lib` was added just for testing purposes.